src/pages/Profile.tsx

```diff
@@ -1,5 +1,222 @@
+    borderRadius: 20,
+    textTransform: "none",
+    marginLeft: 62,
+    marginTop: 2,
+  },
+  displayName: {
+    fontSize: 30,
+    marginTop: 10,
+    marginBottom: 5,
+    fontWeight: "bold",
+  },
+  usernameDisplay: {
+    fontWeight: "normal",
+    fontSize: 20,
+    color: "#71797E",
+    marginBottom: 15,
+  },
+  menu: { marginTop: 60, marginRight: 83 },
+  tabSelection: {
+    marginLeft: 85,
+  },
+  header: { height: "60px", paddingTop: 10 },
+  tweetCountStyle: { fontWeight: "normal", paddingLeft: 5, fontSize: 18 },
+  followercountbtn: {
+    border: "none",
+    fontSize: 15,
+    padding: 0,
+    backgroundColor: "white",
+    color: "black",
+    textTransform: "none",
+    marginTop: 2,
+  },
+  followingcountbtn: {
+    border: "none",
+    fontSize: 15,
+    padding: 0,
+    marginLeft: 5,
+    color: "black",
+    backgroundColor: "white",
+    textTransform: "none",
+    marginTop: 2,
+  },
+  bioDisplay: {
+    marginBottom: 20,
+  },
+};
+
 const Profile = () => {
-  return <h1 style={{ margin: "auto" }}>Profile</h1>;
+  const [value, setValue] = React.useState("one");
+  const [tweetCount, setTweetCount] = React.useState();
+  const [bio, setBio] = React.useState();
+  const [joinDate, setJoinDate] = React.useState("YYYY-MM-DD");
+  const user = useAppSelector((state) => state.user);
+  const handleChange = (event: React.SyntheticEvent, newValue: string) => {
+    setValue(newValue);
+    console.log(value);
+  };
+
+  useEffect(() => {
+    const fetchTweetCount = async () => {
+      const result = await axios.get(
+        "http://localhost:3001/api/profile/getTweetCount",
+        {
+          params: {
+            userId: user.userId,
+          },
+        }
+      );
+      setTweetCount(result.data);
+    };
+
+    const fetchBio = async () => {
+      const result = await axios.get(
+        "http://localhost:3001/api/profile/getBio",
+        {
+          params: {
+            userId: user.userId,
+          },
+        }
+      );
+      setBio(result.data);
+    };
+
+    const fetchJoinDate = async () => {
+      const result = await axios.get(
+        "http://localhost:3001/api/profile/getJoinDate",
+        {
+          params: {
+            userId: user.userId,
+          },
+        }
+      );
+      setJoinDate(result.data);
+    };
+    fetchBio();
+    fetchTweetCount();
+    fetchJoinDate();
+  }, [value, user]);
+  return (
+    <>
+      <Box width={650}>
+        <div style={styles.rightBound}>
+          {/**Not sure about this */}
+          <div style={styles.header}>
+            {" "}
+            <Typography sx={styles.arrowText}>{user.username}</Typography>
+            <button style={styles.arrowBtn}>
+              <ArrowBackIcon></ArrowBackIcon>
+            </button>
+            <Typography sx={styles.tweetCountStyle}>
+              {tweetCount} Tweets
+            </Typography>
+          </div>
+          <div style={{ position: "relative" }}>
+            <img
```

Comment:
I believe this can also be a `Box` component with a variant of `img`. 

Take a look at this: https://stackoverflow.com/questions/61263669/does-material-ui-have-an-image-component